src/main/java/io/minio/MinioClient.java

```diff
@@ -2572,11 +2574,7 @@ private void completeMultipart(String bucketName, String objectName, String uplo
       if (scanner.hasNext()) {
         bodyContent = scanner.next();
       }
-    } catch (EOFException e) {
```

Comment:
I see the change from EOFException to ConnectException.  Is it required to catch ConnectException and handle here?